odoorpc/report.py

```diff
@@ -114,7 +114,12 @@ def check_report(name):
         if v(self._odoo.version)[0] >= 11:
             IrReport = self._odoo.env['ir.actions.report']
             report = IrReport.browse(report_id)
-            response = report.with_context(context).render(ids, data=datas)
+            if v(self._odoo.version)[0] >= 14:
+                response = report.with_context(context)._render(
```

Comment:
For sure it won't work neither as methods prefixed with `_` are not reachable through RPC.
Could be fixed later.